code/ATMOSPHERICS/hvac/spaceheater.dm

```diff
@@ -125,7 +125,9 @@
 
 /obj/machinery/space_heater/campfire/attackby(obj/item/I, mob/user)
 	..()
-	if(!on && cell.charge > 0)
+	var/turf/T = get_turf(src)
+	var/datum/gas_mixture/env = T.return_air()
+	if(!on && cell.charge > 0 && env.oxygen >= 5)
```

Comment:
Yeah. I'm still not gonna miss an opportunity to point out _what the fuck is this_.